src/content/learn/describing-the-ui.md

```diff
@@ -334,19 +334,19 @@ function Item({ name, isPacked }) {
 export default function PackingList() {
   return (
     <section>
-      <h1>Sally Ride's Packing List</h1>
+      <h1>Daftar Perlengkapan Mengemudi Sally</h1>
```

Comment:
```suggestion
      <h1>Daftar Perlengkapan Sally Ride</h1>
```

Sally Ride is the name of an astronaut.

https://en.wikipedia.org/wiki/Sally_Ride